components/gantry/oneaxis/oneaxis.go

```diff
@@ -412,9 +479,12 @@ func (g *oneAxis) Stop(ctx context.Context, extra map[string]interface{}) error
 
 // Close calls stop.
 func (g *oneAxis) Close(ctx context.Context) error {
-	g.mu.Lock()
-	defer g.mu.Unlock()
```

Comment:
I'm open to disagreement on this, but naively I'd expect to need to lock the mutex every time we want to interact with the motor. If this really should be removed, put in a comment in the code explaining why we shouldn't lock the mutex (which will dissuade us from adding it back 6 months from now when we've forgotten about this PR).